websites/api-docs-smoke-test/source-raml/test/api.raml

```diff
@@ -187,25 +209,32 @@ uses:
           fuzzy?:
             type: boolean
             description: Whether to apply fuzzy search on the text to analyze.
+            example: true
           fuzzyLevel?:
             type: number
             description: |
               Provide explicitly the fuzzy level desired if fuzzy is enabled. This value can not be higher than the one
               chosen by the platform by default.
+            example: 7
           staged?:
             type: boolean
             description: Whether to query for the current or staged projections.
+            example: true
           filter?:
             type: string
+            example: bodyMassIndex
           filter.facets?:
             type: string
+            example: weight,height
           filter.query?:
             type: string
+            example: lastName = Doe
           facet?:
             type: string
```

Comment:
> For testing an implementation, it would be good to have a test case where one optional query parameter has no example set. maybe this one?

This has no example.